Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -87,6 +87,7 @@ LogManager::LogManager()
   m_Log[LogTypes::SP1] = new LogContainer("SP1", "Serial Port 1");
   m_Log[LogTypes::VIDEO] = new LogContainer("Video", "Video Backend");
   m_Log[LogTypes::VIDEOINTERFACE] = new LogContainer("VI", "Video Interface (VI)");
+  m_Log[LogTypes::VIDEOCUSTOMTEX] = new LogContainer("VCT", "Video Custom Textures (VCT)");
```

Comment:
I agree that VCT is not obvious moniker, I realize now it's the tag used in practise, in the console output. 

Is anyone in favor of dropping the "video" moniker ?